docs/playbook-reference/actions/scans.rst

```diff
@@ -0,0 +1,48 @@
+Scans
+########
+
+Robusta includes built-in actions to scan and get insights on Kubernetes clusters.
+
+These actions can be triggered automatically on a schedule, manually using the Robusta Platform and :ref:`manually using the robusta cli <Manual Triggers>`.
+
+
+Popeye - A Kubernetes Cluster Sanitizer
+---------------------------------------------
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes cluster and reports potential issues with deployed resources and configurations. by default Robusta will run a Popeye scan on startup.
+
+Popeye scans can be triggerd and viewed in the Robusta platform
+
+.. image:: /images/popeye_example.png
+    :width: 1000
+    :align: center
+
```

Comment:
Can we add a Slack example here:

**With or without the UI, you can configure additional scans on a schedule. The results can be sent as a PDF to Slack, or to the Robusta UI. Other sinks like MSTeams are not supported yet.**

<Show yaml config>